packages/website/plugins/generated-rule-docs.ts

```diff
@@ -243,7 +253,10 @@ export const generatedRuleDocs: Plugin = () => {
 
       optionsH2Index += 2;
 
-      if (meta.schema.length === 0) {
+      const hasConfig = Array.isArray(meta.schema)
+        ? meta.schema.length === 0
+        : Object.keys(meta.schema).length === 0;
+      if (!hasConfig) {
```

Comment:
this condition is not correct, `hasConfig` should be changed to `noConfig` or condition should be changed

```suggestion
      const noConfig = Array.isArray(meta.schema)
        ? meta.schema.length === 0
        : Object.keys(meta.schema).length === 0;
      if (noConfig) {
```

```suggestion
      const hasConfig = Array.isArray(meta.schema)
        ? meta.schema.length > 0
        : Object.keys(meta.schema).length > 0;
      if (!hasConfig) {
```

Reply:
Oops. I extracted the logic when I patched the json types package to fix a type error and I got it wrong.